test/getUriParameters-test.coffee

```diff
@@ -94,17 +107,44 @@ describe('Transformation • Refract • getUriParameters' , ->
             'type': 'enum',
```

Comment:
We need to take the type from the content. It is the only way for now since Parameters is not full MSON yet.